dashboard/src/components/AppView/AppView.tsx

```diff
@@ -109,30 +109,36 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
     const services: Array<IKubeItem<IResource>> = [];
     const ingresses: Array<IKubeItem<IResource>> = [];
     const otherResources: IResource[] = [];
-    manifest.forEach((i: IResource) => {
-      const resource = { isFetching: true, item: i };
+    manifest.forEach((i: IResource | IK8sList<IResource, {}>) => {
+      const item = i as IResource;
+      const resource = { isFetching: true, item };
       switch (i.kind) {
         case "Deployment":
           deployments.push(resource);
           sockets.push(
-            this.getSocket("deployments", i.apiVersion, i.metadata.name, newApp.namespace),
+            this.getSocket("deployments", i.apiVersion, item.metadata.name, newApp.namespace),
           );
           break;
         case "Service":
           services.push(resource);
-          sockets.push(this.getSocket("services", i.apiVersion, i.metadata.name, newApp.namespace));
+          sockets.push(
+            this.getSocket("services", i.apiVersion, item.metadata.name, newApp.namespace),
+          );
           break;
         case "Ingress":
           ingresses.push(resource);
           sockets.push(
-            this.getSocket("ingresses", i.apiVersion, i.metadata.name, newApp.namespace),
+            this.getSocket("ingresses", i.apiVersion, item.metadata.name, newApp.namespace),
           );
           break;
         case "Secret":
-          secretNames.push(i.metadata.name);
+          secretNames.push(item.metadata.name);
+          break;
+        case "List":
+          (i as IK8sList<IResource, {}>).items.forEach(listItem => otherResources.push(listItem));
```

Comment:
hmm what if the List contains a Secret or Service or Deployment? We could extract this outer forEach block to a function and call it recursively in the case of a List

Reply:
👍 I thought that it was not possible to use those type of resources as `List` but I guess there is nothing that prevent that from happening.